scripts/vaccination_alerts/utils.ts

```diff
@@ -141,3 +151,88 @@ export function getUserLocationsToAlert(
     .intersection(locationsToAlert)
     .value();
 }
+
+const templatePath = path.join(__dirname, 'vaccination-alert-template.html');
+
+const alertTemplate = Handlebars.compile(fs.readFileSync(templatePath, 'utf8'));
+
+const colors = {
+  GREY_0: COLOR_MAP.GREY_0,
+  GREY_1: COLOR_MAP.GREY_1,
+  GREY_4: COLOR_MAP.GREY_4,
+  GREY_5: COLOR_MAP.GREY_5,
+  GREEN: COLOR_MAP.GREEN.BASE,
+  LIGHT_YELLOW: 'rgba(255, 201, 0, 0.12)',
+  PURPLE: '#5900EA',
+};
+
+export function generateEmailContent(
+  vaccinationInfo: RegionVaccinePhaseInfo,
+  vaccinationLinks: RegionVaccinationInfo | null,
+): string {
+  const fipsCode = vaccinationInfo.fips;
+  const region = regions.findByFipsCodeStrict(fipsCode);
+
+  const currentPhases = vaccinationInfo.phaseGroups
+    .filter(phaseGroup => phaseGroup.currentlyEligible)
+    .map((phaseGroup, index, allCurrentPhases) => {
+      const isMostRecentlyEligible = index === allCurrentPhases.length - 1;
+
+      return {
+        title: phaseGroup.tier
+          ? `${phaseGroup.phase}, ${phaseGroup.tier}`
+          : phaseGroup.phase,
+        subtitle: isMostRecentlyEligible
+          ? 'most recently eligible'
+          : 'eligible',
+        description: markdownToHtml(phaseGroup.description),
+        isCurrentPhase: isMostRecentlyEligible,
+      };
+    });
+
+  const mostRecentPhase = _.last(currentPhases);
+  const title = mostRecentPhase?.title
+    ? `${region.fullName} is now in ${mostRecentPhase.title} of vaccination`
+    : `${region.fullName} has updated vaccination information`;
+
+  const data = {
+    title,
+    subtitle: 'People who are eligible to be vaccinated now include:',
+    sourceName: `${region.fullName} Health Department`,
+    sourceUrl: vaccinationInfo.eligibilityInfoUrl,
+    eligibilityUrl: vaccinationLinks?.eligibilityInfoUrl,
+    locationName: region.fullName,
+    currentPhases,
+    colors,
+  };
```

Comment:
note to self: I think we should probably move the function that generates the data to send in the emails closer to the CMS, since all the information is taken from there